Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.3
+RUN apt-get update
+RUN apt-get install -y wget build-essential
+
+# Gearcmd
+RUN wget https://github.com/Clever/gearcmd/releases/download/v0.3.1/gearcmd-v0.3.1-linux-amd64.tar.gz
```

Comment:
Can be one line: https://github.com/Clever/powerschool/blob/master/Dockerfile#L15